helm-chart/renku-ui/templates/deployment-template.yaml

```
@@ -55,16 +55,18 @@ spec:
             - name: STATUSPAGE_ID
               value: {{ .Values.statuspage.id | quote }}
             {{- end }}
-              {{- if .Values.sentry.enabled }}
+            {{- if .Values.sentry.enabled }}
             - name: SENTRY_URL
               value: {{ .Values.sentry.url | quote }}
             - name: SENTRY_NAMESPACE
               value: {{ .Values.sentry.namespace | default (printf "%s" .Release.Namespace) | quote }}
-              {{- end }}
-              {{- if .Values.maintenance }}
+            - name: SENTRY_SAMPLE_RATE
+              value: { { .Values.sentry.sampleRate | quote } }
```

```suggestion
              value: {{ .Values.sentry.sampleRate | quote }}
```
I think this is throwing the error in the deployment.